modules/sovrnBidAdapter.js

```diff
@@ -25,11 +27,21 @@ export const spec = {
       let sovrnImps = [];
       let iv;
       let schain;
-      let unifiedID;
+      let eids;
+      let tpid = []
+      let criteoId;
 
       utils._each(bidReqs, function (bid) {
-        if (!unifiedID) {
-          unifiedID = utils.deepAccess(bid, 'userId.tdid');
+        if (!eids && bid.userId) {
+          eids = createEidsArray(bid.userId)
+          eids.forEach(function (id) {
+            if (id.uids && id.uids[0]) {
```

Comment:
`id.uids` is only ever an array with a max of one item in it? Or that's all we care about for now?

Reply:
So in a great moment for standards, I am unsure what to do with a second Id value if it is in the tpid object.  Everyone should be using the eids array but we saw some documentation that indicated that for some ids the tpid was used as well.  My guess is that eventually, everyone will move off of tpid (if anyone is using it now) and we will stop sending it.  Along with the weird, criteo id entry.